mirror-service/src/main/java/org/hyades/osv/OsvMirrorHandler.java

```diff
@@ -58,16 +62,15 @@ private void unzipFolder(ZipInputStream zipIn, String ecosystem) throws IOExcept
             while ((line = reader.readLine()) != null) {
                 out.append(line);
             }
-            JSONObject json = new JSONObject(out.toString());
-            final OsvToCyclonedxParser parser = new OsvToCyclonedxParser();
-            final Bom osvAdvisory = parser.parse(json);
+            var json = new JSONObject(out.toString());
+            Bom osvAdvisory = new OsvToCyclonedxParser(this.objectMapper).parse(json);
             if (osvAdvisory != null) {
                 osvAdvisories.add(osvAdvisory);
             }
             zipEntry = zipIn.getNextEntry();
             reader = new BufferedReader(new InputStreamReader(zipIn));
         }
         reader.close();
-        LOGGER.info(osvAdvisories.size() +" advisories mirrored successfully for ecosystem: "+ ecosystem);
```

Comment:
this log seems useful

Reply:
It was getting logged at 2 places. I usually log info level logs only in public methods. 
This is logged already here I think           
 LOGGER.info("OSV mirroring completed for ecosystem: {} advisories: {}",  ecosystem, osvAdvisories);

Reply:
okay :)